build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-multiplatform.gradle.kts

```diff
@@ -46,6 +46,12 @@ kotlin {
         binaries.executable()
     }
 
+    @OptIn(org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl::class)
+    wasmWasi {
+        nodejs()
+        binaries.executable()
```

Comment:
@fzhinkin sorry for bothering again, can I ask why you explicitly selected an executable binary?  
Wouldn't this be done by consumers of the library?

Reply:
@lppedd thanks for pointing to this! You're absolutely right, it should be done by consumers and there's absolutely no sense in specifying it for the library. I'll remove it.